snippets/csharp/VS_Snippets_CLR_System/system.Math.Abs/cs/Abs1.cs

```diff
@@ -1,22 +1,21 @@
-﻿// <Snippet1>
-using System;
+﻿using System;
 
 public class Example
 {
    public static void Main()
    {
+      // <Snippet1>
       decimal[] decimals = { Decimal.MaxValue, 12.45M, 0M, -19.69M, 
                              Decimal.MinValue };
       foreach (decimal value in decimals)
          Console.WriteLine("Abs({0}) = {1}", value, Math.Abs(value));
```

Comment:
You may check if there are other places where string interpolation can be used.

```suggestion
         Console.WriteLine($"Abs({value}) = {Math.Abs(value)}");
```

Reply:
This set of changes is optional, but highly recommended.

Reply:
This instance is fixed -- looking for others now.

Reply:
@BillWagner, @Youssef1313 -- there are 13 more such constructs in this set of example files; working on them.